Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -692,6 +692,58 @@ of issues:
   }
 }
 
+### Run tests sequentially
+
+By default, the testing library runs all tests in a suite in parallel. The
+default behavior of XCTest is to run each test in a suite sequentially. If your tests use
+shared state, for example, global variables, you may see expected behavior
+including unreliable test outcomes when you run tests in parallel.
+
+Annotate your test suite with ``Trait/serialized`` to run tests within that
+suite serially:
+
+@Row {
+  @Column {
+    ```swift
+    // Before
+    class RefrigeratorTests : XCTestCase
+      func testLightComesOn() async {
```

Comment:
These functions are all marked `async`, but none of them uses `await` and all of them use `try`. Did you mean to mark them `throws` instead?

Reply:
Actually I just followed the existing examples in this article, for example the one in "Annotate known issues" that starts at line 574. Should we take a look at all of the examples in this article and decide what their signatures should be?